classes/plugins/importexport/PKPImportExportDeployment.inc.php

```diff
@@ -213,8 +213,13 @@ function setFileDBIds($fileDBIds) {
 	 * @return integer
 	 */
 	function getFileDBId($fileId, $revisionId) {
```

Comment:
I realized that we should consider revisionId parameter = null and return null in case the revisionId parameter is not null and DBId not found. This would check if that specific revision is inserted. If revisionId parameter is null and fileId exists the current DBId should be returned. Thus I will change it so...